src/gui/widgets/scroll_text.hpp

```diff
@@ -163,6 +165,38 @@ class scroll_text : public scrollbar_container
 
 	void place(const point& origin, const point& size);
 
+	/** See @ref widget::calculate_best_size. */
+	point calculate_best_size() const override
+	{
+		point calc_size = scrollbar_container::calculate_best_size();
+
+		if ((calc_size.x > max_size_.x) && (max_size_.x != 0)) {
+			calc_size.x = max_size_.x;
+		}
+
+		if ((calc_size.y > max_size_.y) && (max_size_.y != 0)) {
+			calc_size.y = max_size_.y;
+		}
+
+		return calc_size;
+	}
+
+	void set_max_size(point max_size)
+	{
+		/***** get vertical scrollbar size *****/
```

Comment:
Using `/**` implies a Doxygen comment, which shouldn't apply to stack variables.

Also, I think both of these function implementations should be in the .cpp file rather than the .hpp file.

Reply:
Changed to C++ style comment.

Reply:
The implementations of the two methods have been moved to the cpp file.